gbus/tx/mysql/migrations.go

```diff
@@ -63,6 +63,22 @@ func sagaStoreAddRPCIDDetails(svcName string) *migrator.Migration {
 	}
 }
 
+func sagaStoreAddCreatedAtDetails(svcName string) *migrator.Migration {
+	tblName := tx.GrabbitTableNameTemplate(svcName, "sagas")
+
+	addCreatorDetailsSQL := `ALTER TABLE ` + tblName + ` ADD COLUMN created_at DATETIME NOT NULL DEFAULT NOW() AFTER version`
```

Comment:
@kerenbe4  I am not sure whether the migration will run properly if there are already instances in the database as the column is defined as NOT NULL.
It may add the DEFAULT NOW() value when creating the new column on existing records but I am not sure.
Need to verify this.

Reply:
I verified that it adds NOW() on the already existing records 👍🏼